src/Elastic.Apm/Model/Transaction.cs

```diff
@@ -21,943 +21,945 @@
 using Elastic.Apm.Report;
 using Elastic.Apm.ServerInfo;
 
-namespace Elastic.Apm.Model
+namespace Elastic.Apm.Model;
+
+internal class Transaction : ITransaction
```

Comment:
Changes in this file are minimal, but due to my usual fight with `dotnet format` I hit full reformat on it (after going through multiple fails). Now it seems to be ok, but there are a bunch of formatting changes here. Sorry about that.

Reply:
Reviewing with `?w=1` always helps in these cases :) Let me know if you want to pair over why `dotnet format` is acting up. It might be because your checkout predates some of the cleanup.